AirCasting/MainTabBarView.swift

```diff
@@ -94,8 +97,13 @@ private extension MainTabBarView {
             .tag(TabBarSelection.Tab.dashboard)
             .overlay(
                 Group{
-                    if !reorderButton.isHidden && sessions.count > 1 && selectedSection.selectedSection == .following {
-                        reorderingButton
+                    HStack {
+                        if !searchAndFollow.isHidden && featureFlagsViewModel.enabledFeatures.contains(.searchAndFollow) {
```

Comment:
I think maybe this button also should only be present in following tab? Do you know if that's not what Michael wanted?

Reply:
I thought that it should be available all the time, but I will ask and clarify. Thank you!